src/main/java/cn/nukkit/block/BlockBamboo.java

```diff
@@ -55,7 +55,7 @@ public int onUpdate(int type) {
             level.useBreakOn(this, null, null, true);
         } else if (type == Level.BLOCK_UPDATE_RANDOM) {
             Block up = up();
-            if (getAge() == 0 && up.getId() == AIR && level.getFullLight(up) >= BlockCrops.MINIMUM_LIGHT_LEVEL && ThreadLocalRandom.current().nextInt(3) == 0) {
```

Comment:
what is `upisAir()`

Reply:
Forget to add '.' between up and isAir()